test/functional/sortTemplate.test.ts

```diff
@@ -0,0 +1,310 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// ----------------------------------------------------------------------------
+
+// tslint:disable:no-unused-expression max-func-body-length promise-function-async max-line-length no-http-string no-suspicious-comment
+// tslint:disable:no-non-null-assertion
+
+// WARNING: At the breakpoint, the extension will be in an inactivate state (i.e., if you make changes in the editor, diagnostics,
+//   formatting, etc. will not be updated until you F5 again)
+
+import * as assert from 'assert';
+import * as fse from 'fs-extra';
+import { commands, window, workspace } from "vscode";
+import { getTempFilePath } from "../support/getTempFilePath";
+
+suite("SortTemplate", async (): Promise<void> => {
```

Comment:
Thanks for adding these!  Would love to see some tests specific to:
* multi-line comments
* multi-line strings
* when a specific section doesn't exist (e.g. sort functions when there are none - only need one of these)

Reply:
Added the tests